apps/doji-backend/src/session/session.dto.ts

```diff
@@ -1,15 +1,15 @@
 import { SessionStatus } from '@backend/entities/Session'
 import { User } from '@backend/entities/User'
-import { ReviewStatDTO } from '@backend/review/review.dto'
+import { ReviewStatResponseDTO } from '@backend/review/review.dto'
 import {
   IGetServiceByNameAndExpertUsernameDTO,
-  IPublicSessionReviewDTO,
+  IPublicSessionReviewDTO as IPublicSessionReviewResponseDTO,
   IReviewStat,
   IScheduleSessionDTO,
   IService,
   IServiceInformationDTO,
   ISession,
-  ISessionInformationDTO,
+  ISessionInformationDTO as ISessionInformationResponseDTO,
```

Comment:
name alias 😥

Reply:
Oops. My bad. Auto-rename tools is weird.